src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -50,11 +51,40 @@ public CommandResult execute() {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
             final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            if (!IsDisjointSet(wordsInName, keywords)) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
 
+    /**
+     * Check if both set are disjoint
+     *
+     * @param firstSet
+     * @param secondSet
+     * @return true if two both set are disjoint
+     */
+    private boolean IsDisjointSet(Set<String> firstSet, Set<String> secondSet) {
+        return Collections.disjoint(toLower(firstSet), toLower(secondSet));
+    }
+
+    /**
+     * Function to convert a set of string values to all small letters
+     *
+     * @param strings
+     * @return orignal set with all string to lowercase
+     */
+    private Set<String> toLower(Set<String> strings) {
+        String[] stringsArray = strings.toArray(new String[0]);
+        Set<String> returnString = new HashSet<>();
+
+        for (int i=0; i<stringsArray.length; ++i) {
```

Comment:
Minor suggestion: do keep in mind the spacing in expressions (ex: should be `i = 0`).